build.sh

```diff
@@ -0,0 +1,161 @@
+  echo USAGE:
+  echo   $0 [/path/to/installation/dir]
+  echo IMPORTANT: Please enter an empty or new directory as installation directory.
+  exit 1
+}
+
+################################################################################
+function checkParameters {
+################################################################################
+  # Check no of parameters.
+  if [ "$#" -ne 1 ]; then
+    echo "Illegal number of parameters"
+    usage
+  fi
+
+  # Check if argument is given
+  if [ -z "$1" ]; then
+    echo Please provide a directory where to install.
+    usage
+    exit 1
+  fi
+  
+  # Check for invalid flags
+  if [ "${1:0:1}" = "-" ]; then
+    usage
+  fi
+
+  INSTALLATION_DIRECTORY=$1
+
+  # Check if directory exists
+  if [ ! -d "$INSTALLATION_DIRECTORY" ]; then
+    # Create directory if it doesn't exists.
+    mkdir -p "$INSTALLATION_DIRECTORY"
+    if [ $? -ne 0 ]; then
+      echo "Error creating directory $INSTALLATION_DIRECTORY!"
+      exit 1
+    fi
+  fi
+  # Check if directory is empty
+  if [ ! -z "$(ls -A "$INSTALLATION_DIRECTORY")" ]; then
+     echo $INSTALLATION_DIRECTORY is not empty!
+     echo "Please enter an empty or a new directory!"
+     exit 1
+  fi
+}
+
+################################################################################
+function printInfo {
+################################################################################
+echo ---------------------------------------------------------------------------
+echo $*
+echo ---------------------------------------------------------------------------
+}
+
+################################################################################
+# END DECLARATION FUNCTIONS / START OF SCRIPT
+################################################################################
+
+################################################################################
+# Test for commands used in this script
+################################################################################
+testForCommands="dirname git date java javac"
+
+for command in $testForCommands
+do 
+  type $command >> /dev/null
+  if [ $? -ne 0 ]; then
+    echo "Error: command '$command' is not installed!"
+    exit 1
+  fi
+done
+
+################################################################################
+# Determine directory of script. 
+################################################################################
+ACTUAL_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+################################################################################
+# Check parameters
+################################################################################
+checkParameters $*
+
+################################################################################
+# Determine repo name 
+################################################################################
+REPO_URL=`git remote get-url origin`
+REPO_NAME=${REPO_URL##*/}
```

Comment:
If a repository was cloned with it's .git suffix, it must be removed to find the jar file later on. With the following change, I was then able to run the container and access the swagger interface successfully:

```suggestion
REPO_NAME=${REPO_URL##*/}
REPO_NAME=${REPO_NAME%".git"}
```